src/pyflex/window_selector.py

```diff
@@ -184,6 +184,21 @@ def default(self, obj):
             except TypeError:
                 filename.write(j.encode())
 
+    def write_text(self, filename):
```

Comment:
Whats the motivation for this function? I.e. when would I want to use this instead of the JSON serialization? I'm not against having it - just wundering.

This function also has to be added to the tests.